spec/models/library_entry_spec.rb

```diff
@@ -134,6 +135,49 @@
     end
   end
 
+  describe 'timestamp validation' do
+    let!(:anime) { create(:anime) }
```

Comment:
you generally don't have `let!` statements inside a describe block, normally just used at top of file. If you need specific for multiple `it` blocks you can do a `before(:each)` block.